app/helpers/storytime/application_helper.rb

```diff
@@ -27,6 +27,14 @@ def active_nav_item_class(controller, type = nil)
       'class="active"'.html_safe if controller == current_controller
     end
 
+    def active_blog_item_class(blog)
+      path_arr = request.path.split("/")
+
+      return unless path_arr.include?("blogs")
```

Comment:
Is it possible to do this based on the controller & id params? I feel like that'd be a little less brittle.
```'class="active".html_safe if params[:controller] == "storytime/dashboard/blogs_controller" && params[:id] == blog.slug```

maybe?